safe_webrtc_example/app/constants.js

```diff
@@ -0,0 +1,74 @@
+export default {
+  UI: {
+    DEFAULT_LOADING_DESC: 'Please wait...',
+    CONN_MSGS: {
+      INIT: 'Initialising connection',
+      SEND_INVITE: 'Invite sent. Waiting for the remote peer to accept the connection',
+      INVITE_ACCEPTED: 'Invite accepted. Establishing connection with remote',
+      CALLING: 'Remote accepted the invite. Establishing connection with remote',
+    },
+    CONN_TIMER_INTERVAL: 2000,
+    TIMER_INTERVAL: {
+      FETCH_INVITES_POLL: 5000,
+      CONNECTION_POLL: 4000,
+    },
+  },
+  CONFIG: {
+    SERVER: {
+      iceServers: [
+        { url: 'stun:stun1.l.google.com:19302' }, // URL to STUN Server
```

Comment:
Perhaps replace the value with a placeholder similar to how it's done with the credential and username.